### PR TITLE
update storage class.md

### PR DESCRIPTION
**Pull request by akshayamadhuri**
update storage class.md
issue number #41821

### CODE REVIEW

File: content/en/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -46,10 +49,10 @@ for details.
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
-  name: standard
+  name: sc-name
```
Comment (sftim) [inline content/en/docs/concepts/storage/storage-classes.md:52]: `sc-name` isn't a particularly good name for a StorageClass; why change it?

File: content/en/docs/concepts/storage/storage-classes.md [hunk 2]
```diff
@@ -46,10 +49,10 @@ for details.
...
 provisioner: kubernetes.io/aws-ebs
 parameters:
-  type: gp2
+  pools: pool1,pool2,pool3
```
Comment (sftim) [inline content/en/docs/concepts/storage/storage-classes.md:55]: Is this a parameter that the `kubernetes.io/aws-ebs` storage driver understands?

**Review by sftim — COMMENTED** [review]
Thanks.

However, this change doesn't look right.

File: content/en/docs/concepts/storage/storage-classes.md (hunk 2 above)
Comment (akshayamadhuri) [reply content/en/docs/concepts/storage/storage-classes.md:55]: i think the issue needed to be adding the pool should i raise a PR with documentation changes without yaml file

**Comment by sftim** [comment]
@akshayamadhuri, please write a PR description that summarised why we should make this change.

**Comment by chenzhiwei** [comment]
storageclass is not an immutable object, but the current description said it can not be updated once created.

the parameters and the volume expansion attributes can be updated and there are some use cases.

1. the csi driver initially does not support volume expansion feature, but later it supports. then users need to update the storageclass to set allow volume expansion to true.
2. the storage pools can be set in parameters.pools, initially users only have one pool, later they added a new pool to increase the capacity, so they need to update the storage class to reflect this.

**Comment by reylejano** [comment]
Hi @akshayamadhuri , I'm closing this PR as reviewers have commented that the changes proposed in this PR are correct and this PR has a merge conflict and needs rebase
If you feel otherwise, feel free to reopen the PR with `/reopen` in a comment
/close

**Pull request closed without merging**